src/app/launcher/service/project-summary.service.ts

```diff
@@ -12,7 +13,14 @@ export abstract class ProjectSummaryService {
    * @param {Summary} summary The project summary
    * @returns {Observable<boolean>}
    */
-  abstract setup(summary: Summary): Observable<boolean>;
+  abstract setup(summary: Summary, spaceId: string, spaceName: string): Observable<boolean>;
+
+  /**
+   * Get the current context details
+   *
+   * @returns {Observable<Context>}
+   */
+  abstract getCurrentContext(): Observable<Context>;
```

Comment:
The Context object of ngx-fabric8-wit typically provides information about the space the user currently is in. For OSIO, the app-launcher component is shown in the context of a space, otherwise the space path is undefined. 

For the Launcher workflow, does the user have a space? I'm just wondering what Context would provide in that scenario? 

I suspect the service for Launcher will have to create its own Context object?

Reply:
@dlabrecq in the app launcher component, i currently get the context details from ngx-fabric8-wit. However if you have some better approach for this , please let me know

Reply:
That's fine. Just double checking a space available for the Launcher work flow.

Reply:
thanks for the reply